configs/bokeh.json

```
@@ -1,35 +1,88 @@
 {
   "index_name": "bokeh",
   "start_urls": [
-    "https://docs.bokeh.org"
+    {
+      "url": "https://docs.bokeh.org/(?P<lang>.*?)/(?P<version>.*?)/user_guide/",
+      "selectors_key": "user_guide",
+      "page_range": 3,
+      "variables": {
+        "lang": ["en"],
+        "version": ["dev", "latest", "1.3.4", "1.2.0", "1.1.0", "1.0.4", "0.12.16", "0.12.15", "0.12.14", "0.11.0"]
+      }
+    },
+    {
+      "url": "https://docs.bokeh.org/(?P<lang>.*?)/(?P<version>.*?)/reference/",
+      "selectors_key": "reference",
+      "page_range": 2,
+      "variables": {
+        "lang": ["en"],
+        "version": ["dev", "latest", "1.3.4", "1.2.0", "1.1.0", "1.0.4", "0.12.16", "0.12.15", "0.12.14", "0.11.0"]
+      }
+    },
+    {
+      "url": "https://docs.bokeh.org/(?P<lang>.*?)/(?P<version>.*?)/dev_guide/",
+      "selectors_key": "dev_guide",
+      "page_range": 1,
+      "variables": {
+        "lang": ["en"],
+        "version": ["dev", "latest", "1.3.4", "1.2.0", "1.1.0", "1.0.4", "0.12.16", "0.12.15", "0.12.14", "0.11.0"]
+      }
+    }
   ],
   "stop_urls": [
     "/_",
     "/docs/gallery/"
   ],
   "selectors": {
-    "lvl0": {
-      "selector": ".current .toctree-l1:nth-child(2)",
-      "global": true,
-      "default_value": "Documentation"
+    "strip_chars": "¶",
+    "default": {
+      "lvl0": {
+        "selector": ".current .toctree-l1:nth-child(2)",
+        "global": true,
+        "default_value": "Documentation"
+      },
+      "lvl1": "title",
+      "lvl2": ".content h2, .class > dt .descname",
+      "lvl3": ".content h3, .method .descname",
+      "text": ".content p, .content li"
+    },
+    "user_guide": {
+      "lvl0": {
+        "selector": "#junk",
```

Good guess. Using an empty selector will be more explicit
```suggestion
        "selector": "",
```